### PR TITLE
Use floats instead of decimals for temperature measurements

### LINKED ISSUE #27: Type of temperature measurement
A temperature measurement currently looks like this:

```javascript
    {
        "created_at": "2017-05-01T21:49:38.971Z",
        "custom_attributes": null,
        "id": 57,
        "sensor_id": 3,
        "temperature": "25.5625",
        "url": "https://watertemp-api.coredump.ch/api/measurements/57"
    },
```

Is there a reason why the temperature is a string? Why not a float?

### PR DESCRIPTION
**Pull request by schmijos**
Fixes #27

### CODE REVIEW

File: app/controllers/measurements_controller.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def aggregated
     @measurements = @measurements.group('DATE(created_at)').reorder('DATE(created_at)')
     @minimum_temperature = @measurements.minimum(:temperature)
     @maximum_temperature = @measurements.maximum(:temperature)
-    @average_temperature = @measurements.average(:temperature)
+    @average_temperature = @measurements.average(:temperature).transform_values(&:to_f)
```
Comment (schmijos) [inline app/controllers/measurements_controller.rb:20]: Well… this workaround results out of a 12 years old specialty of `AVG`: https://github.com/rails/rails/pull/40351

Comment (dbrgn) [reply app/controllers/measurements_controller.rb:20]: Haha, nice work digging into that old code :slightly_smiling_face:

**Review by dbrgn — APPROVED** [review]
Looks good to me! This is a breaking change, but it should be quick to fix. We don't need a workaround for the website (we can simply deploy the new version), but I wonder if we should first add code to the Android app to support both variants (string and float).

**Comment by dbrgn** [comment]
Note: It seems that the Android app currently "magically" parses the string as float thanks to the Gson parser. That means that no adjustments will be necessary, as far as I can tell.

The fix for the website should be a one-liner.

**Comment by dbrgn** [comment]
@schmijos thanks, I'll try to deploy API and website tonight.

@Makram95 the iOS app might need adjustments to handle the floats.

**Pull request merged**